### PR TITLE
Add dataset cards to methodology page

### PR DESCRIPTION
**Pull request by vim-usds**
This is just an initial draft to get some quick feedback from design.

### CODE REVIEW

File: client/src/components/datasetCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.datasetCard {
+  background-color: white;
+  padding: 2.7rem 3rem 3rem 3rem;
```
Comment (NatHillardUSDS) [inline client/src/components/datasetCard.module.scss:3]: Good call on using `rem` - I was reading about this yesterday and I didn't realize the limitations around using `em`, which is local to its context. `rem` definitely seems superior

File: client/src/components/datasetCard.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <h3 className={styles.datasetCardIndicator}>{datasetCardProps.indicator}</h3>
+      <div className={styles.datasetCardWhatIsIt}>What is it?</div>
+      <div className={styles.datasetCardDescription}>
+        {datasetCardProps.description}
+      </div>
+
+      <ul className={styles.datasetCardList}>
+        <li className={styles.datasetCardListItem}>
+          <span className={styles.datasetCardLabels}>
+            Data resolution:&nbsp;
```
Comment (NatHillardUSDS) [inline client/src/components/datasetCard.tsx:22]: Two questions about this one:
1. Can we move the `nbsp` to css ?
2. Can we use `formatMessage`?

**Review by NatHillardUSDS — APPROVED** [review]
Overall looks good, a few smaller comments that we could optionally address later as part of other tickets

File: client/src/components/datasetCard.module.scss (hunk 1 above)
Comment (vim-usds) [reply client/src/components/datasetCard.module.scss:3]: yeah totally still up in the air on this.. also heard that rem works well!

File: client/src/components/datasetCard.tsx (hunk 2 above)
Comment (vim-usds) [reply client/src/components/datasetCard.tsx:22]: great call on the `nbsp`! and yes to `formatMessage`

Comment (vim-usds) [reply client/src/components/datasetCard.tsx:22]: nbsp = not being super productive (lazy) 🤣

**Comment by vim-usds** [comment]
Draft PR is completed.. time for the real thing!

**Pull request closed without merging**